src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/GoCardlessEventHandlerForMandatesTest.java

```diff
@@ -48,43 +56,52 @@
             .withPayerFixture(payerFixture);
 
     private GoCardlessEventFixture goCardlessEventFixture = GoCardlessEventFixture.aGoCardlessEventFixture()
+            .withResourceType(GoCardlessResourceType.MANDATES)
             .withLinksOrganisation(organisationIdentifier);
 
     @Test
     public void handle_onAFailedMandateGoCardlessEvent_shouldRegisterAPayEventAsMandateFailed() {
```

Comment:
The name of this test (and others) seem to have drifted apart from what’s actually tested, which is sending emails for certain events.

Reply:
fixed

Reply:
renamed to SendEmailsForGoCardlessMandateEventsHandlerTest